ktor-features/ktor-locations/jvm/src/io/ktor/locations/Location.kt

```diff
@@ -231,7 +231,7 @@ public fun <T : Any> Route.handle(dataClass: KClass<T>, body: suspend PipelineCo
  * Retrieves the current call's location or `null` if it is not available (request is not handled by a location class),
  * or not yet available (invoked too early before the locations feature takes place).
  */
-@KtorExperimentalAPI
+@KtorExperimentalLocationsAPI
```

Comment:
Should we introduce a new annotation for locaitons?

Reply:
We already have it, we just accidentally marked it with the wrong annotation. All other locations-related declarations are `@KtorExperimentalLocationsAPI`